app/assets/v2/js/pages/bounty_details.js

```diff
@@ -225,60 +221,67 @@ var callbacks = {
     return [ 'network', warning ];
   },
   'additional_funding_summary': function(key, val, result) {
-    if (typeof val == 'undefined' || Object.keys(val['tokens']).length == 0) {
-      $('.additional_funding_summary').addClass('hidden');
-      return [ 'additional_funding_summary', '' ];
-    }
-    const usd_value = val['usd_value'];
-    const tokens = val['tokens'];
-    const decimals = 3;
-
-    let crowdfunded_tokens = [];
-    let tooltip_info = [];
-
-    crowdfunded_tokens.push({
-      amount: token_value_to_display(result['value_in_token'], decimals),
-      token: result['token_name']
-    });
 
-    tooltip_info.push('<p class="font-weight-bold m-0 pb-1" style="color:#775EC7;">Intial : ' +
-      $('#value_in_token').html() + '</p>');
+    const tokens = Object.keys(val);
 
-    if (usd_value) {
-      tooltip_info.push('<p class="m-0" style="margin-top: 3px;">Crowdfunding worth $' + usd_value + '</p>');
-      $('#value_in_usdt').html(parseFloat(result['value_in_usdt']) + usd_value);
+    if (tokens.length === 0) {
+      return [ key, val ];
     }
-  
-    for (var token in tokens) {
-      if (token) {
-        const val = tokens[token];
-        const funding = normalizeAmount(val, decimals);
-
-        if (crowdfunded_tokens.filter(fund => fund.token === token).length > 0) {
-          crowdfunded_tokens.map((fund, index) => {
-            if (fund.token === token) {
-              crowdfunded_tokens[index].amount += funding;
-            }
-          });
-        } else {
-          crowdfunded_tokens.push({
-            amount: funding,
-            token: token
-          });
-        }
-        const template = '<p class="m-0" style="color:#fb9470">+' + funding + ' ' + token + ' in crowdfunding</p>';
 
-        tooltip_info.push(template);
-      }
+    const decimals = 2;
```

Comment:
We are showing 3 decimal places in almost all places. Any reason why we are showing only 2 for crowdfunded amount?

Reply:
@SaptakS Aesthetics? 😄 Fixed!